File: est datos/java/Tarea_1.java
import java.util.Scanner;
public class Tarea_1{
    public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
        
    System.out.println("Ingrese el tamaño del arreglo:");
    int tam = sc.nextInt();
    sc.nextLine(); 
    
    String[] array = new String[tam];

    for (int i=0 ;  i < tam ; i++){
        System.out.println("ingrese el valor de la posicion " + (i+1) + ":");
        array[i] = sc.nextLine();
    }

    System.out.println("Los valores del arreglo son:");
    for (int i = 0 ; i < tam ; i++){
        System.out.println("El valor de la posicion " + (i+1) + " es: " + array[i]);
    }


}
}

File: est datos/c#/Tarea_1.cs
int tam = 0;

Console.WriteLine("Ingresa el tamaño del arreglo: ");
string? input = Console.ReadLine();

if (!int.TryParse(input, out tam) || tam <= 0)
{
    Console.WriteLine("Por favor, ingresa un número entero válido y mayor que cero.");
    return 1;
}

dynamic[] array = new dynamic[tam];

Console.WriteLine("Ingresa los elementos del arreglo: ");

for (int i = 0; i < tam; i++)
{
    Console.WriteLine("Ingrese el elemento # " + (i + 1) + ":");
    array[i] = Console.ReadLine();
}

Console.WriteLine("Los elementos del arreglo son: ");
for (int i=0; i<tam; i++)
{
    Console.WriteLine("El elemento # " + (i + 1) + " es: " + array[i]);
}

return 0;